torch/export/_trace.py

```diff
@@ -1892,7 +1899,17 @@ def _produce_guards_callback(gm):
         )
 
     tx = TracingContext(fake_mode)
-    with fake_mode, _NonStrictTorchFunctionHandler(), tracing(tx):
+
+    # We also need to attach dynamo configs as these will be used in HOOs that
```

Comment:
Are these specifically for control flow ops or invoke_subgraph? For control flow ops, we have a _set_compilation_env to put in the flags that're different from the default. 

This PR also changes the behavior of strict export, right?

Reply:
This shouldn't change the behavior of strict export. Those flags are already set in [dynamo export](https://fburl.com/code/5gtic5cg).

These flags directly affect some testing I was doing with invoke_subgraph, but I feel like they would also help with stuff in cond?